src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetProjectManagerService.cs

```diff
@@ -441,7 +441,9 @@ public async ValueTask ExecuteActionsAsync(IReadOnlyList<ProjectAction> actions,
             DependencyBehavior dependencyBehavior,
             IReadOnlyList<string> packageSourceNames,
             VersionRange? versionRange,
-            CancellationToken cancellationToken)
+            CancellationToken cancellationToken,
+            string? newMappingID = null,
+            string? newMappingSource = null)
```

Comment:
I think I commented elsewhere already, but CA1068 usually gets us to put CancellationTokens as the last method parameter. I don't know why this analyzer isn't running for us.